src/components/SignIn.js

```diff
@@ -90,6 +90,7 @@ export default class SignIn extends React.Component {
               width={1}
             />
             <StyledButton
+              id="signin"
```

Comment:
Using `id` is not the way to go to target elements in cypress. We try to get rid of these as it's not a good practice: if we try to show two `SignIn` components at different places in the same page, this will create an error.

As per https://docs.opencollective.com/help/developers/testing-with-cypress#use-data-cy-to-target-dom-elements, the way to go is to use the `data-cy` attribute.

```suggestion
              data-cy="signin-btn"
```

An example of how to use it in a test: https://github.com/opencollective/opencollective-frontend/blob/63294b8a27dee05159042a3694e9ce9e5438df17/cypress/integration/14-contributionFlow.success.test.js#L39

Reply:
Done